src/controllers/handlers/name-owner-handler.ts

```diff
@@ -0,0 +1,14 @@
+import { HandlerContextWithPath, NameOwner } from '../../types'
+
+export async function nameOwnerHandler(
+  context: HandlerContextWithPath<'nameOwnerFetcher' | 'logs', '/users/names/:name/owner'>
+): Promise<{ status: 200; body: NameOwner }> {
+  const { name } = context.params
+  const { nameOwnerFetcher } = context.components
+
+  const page = await nameOwnerFetcher.fetchOwnedElements(name)
```

Comment:
What do you think about returning a 404 if the name does not exist (it has no owner)?